### PR TITLE
Issue500 eq air temp

### LINKED ISSUE #500: Clean up equivalent air temperature calculations
In contrast to TRY weather data sets, TMY sets do not provide data about environmental radiation (which is not atmospheric radiation but in most cases exactly the same values). The `equivalentAirTemperature` models account for this by omitting the calculation parts given in VDI 6007-1 dealing with environmental radiation. When reviewing the models for integration in `AixLib`, I found some relicts of these calculations in the actual Annex60 models. I will withdraw these relicts (that lead to "correction" factors around 1).

### PR DESCRIPTION
**Pull request by mlauster**
This closes #500 

@mwetter, could you run the unit tests? Changes in the results should be minimal.

### CODE REVIEW

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo [hunk 1]
```diff
@@ -65,6 +65,18 @@ initial equation
    equivalent air temperature calculation is close to 0.
    If there are no walls, windows and ground at all, this might be
    irrelevant.", level=AssertionLevel.warning);
+
+equation
+  delTEqLW=(TBlaSky-TDryBul)*alphaRad/(alphaRad+alphaWallOut);
+  delTEqSW=HSol*aExt/(alphaRad+alphaWallOut);
+  if withLongwave then
+    TEqWin=TDryBul.+delTEqLWWin*abs(sunblind.-1);
```
Comment (mwetter) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: If `0 <= sunblind <= 1` according to the comment, why don't you simply use
```
TEqWin=TDryBul.+delTEqLWWin*(1.-sunblind);
```
Also, please use
```
 Modelica.Blocks.Interfaces.RealInput sunblind[n](
   each min=0,
   each max=1,
   each final unit="1")
    "Opening factor of sunblinds for each direction (0 - open to 1 - closed)"
```

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo [hunk 2]
```diff
@@ -106,6 +118,15 @@ initial equation
   revisions="<html>
   <ul>
   <li>
+  September 26, 2016, by Moritz Lauster:<br/>
+  Removed deprecated parameters and values 0.93 and eExt.<br/>
+  Renamed alphaRadWall to alphaRad. Deleted alphaRadWin.<br/>
```
Comment (mwetter) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: Enclose variable names in `<code>...</code>`

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo [hunk 3]
```diff
@@ -28,13 +28,26 @@ T<sub>EqAirExt</sub> = T
...
+&alpha;<sub>Rad</sub>/
 (&alpha;<sub>Rad</sub>+&alpha;<sub>Ext</sub>).
 </p>
+<p>
+The Guideline VDI 6007 Part 1 considers in 
+addition an environmental temperature and tilt angles for all orientations. As 
+necessary inputs for the environmental temperature are not defined in TMY weather 
+data sets, the influence of this temperature is not considered in the presented 
+models. It is in any case a minor effect as black-body sky temperature and 
+environmental temperature hardly differ. Furthermore, the Guideline VDI 6007 Part
```
Comment (mwetter) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo]: The "hardly differ" is not true. See `"Annex60.BoundaryConditions.WeatherData.Examples.ReaderTMY3"` in which the difference between dry bulb and black body sky temperature is often around 15 Kelvin, up to 20 Kelvin.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo (hunk 1 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: True, `abs()` is not necessary here. But as `sunblind` is a vector, it ends in:

`TEqWin=TDryBul.+delTEqLWWin*(ones(n)-sunblind);`

So one function call for the other. But it looks more straight ahead.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo (hunk 2 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: Done

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo (hunk 3 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo]: The environmental temperature is not the dry bulb temperature. It's an analog temperature to the black body sky temperature for the environment (radiative temperature of the ground). 

It is calculated similar to `TBlaSky`

`TBlaSky  =  (HHorIR/Modelica.Constants.sigma)^0.25;`

as 

`TBlaEarth  =  (-HEarth/Modelica.Constants.sigma)^0.25;`

`HEarth`and `HHorIR` are both given in TRY weather data sets and are for most time steps identical or differ only slightly.
The VDI 6007 refers to it as temperature due to radiation form the environment. Do you have a better name for it? I will add some lines for explanation to the documentation.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo (hunk 1 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: Done

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo (hunk 3 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo]: Done

**Comment by mlauster** [comment]
@mwetter , I fixed all open issues, could you again review?

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo (hunk 3 above)
Comment (mwetter) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo]: Would it be clearer to call it "environmental radiative temperature"? I understood, maybe mistakenly, by environmental temperature an average between dry-bulb and sky radiative temperature.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo [hunk 4]
```diff
@@ -51,7 +51,10 @@ partial model PartialVDI6007
     displayUnit="degC") "Equivalent air temperature"
     annotation (Placement(
     transformation(extent={{100,-10},{120,10}})));
-  Modelica.Blocks.Interfaces.RealInput sunblind[n]
+  Modelica.Blocks.Interfaces.RealInput sunblind[n](
+    each min=0,
+    each max=0,
```
Comment (mwetter) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: This should be `each max=1`.

**Comment by mwetter** [comment]
@mlauster I added two more comments, and can afterwards run the regression tests again when I have a good network connection.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo (hunk 4 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo]: That's true, done.

File: Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo (hunk 3 above)
Comment (mlauster) [inline Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo]: Good idea, done.

**Comment by mlauster** [comment]
@mwetter , I implemented all your suggestions, might be ready to merge now. :)

**Pull request merged**